### DISCUSSION TITLE
turning off bromite’s grid view for tabs

### DISCUSSION THREAD

**Discussion by Moon1moon** [bromite/bromite#1550] [Ideas]
turning off bromite’s grid view for tabs | There is a parameter about this in flags. But it does not work. The setting: https://youtu.be/rj1QRti_gIg (not working) My opinion chrome and bromite's the tab view a few years ago were better. I think it would be useful to have a optional setting to switch to the old tab view.

**Comment by bagajohny** [comment]
I also want this feature. The Accessibility setting option is also not available in bromite. Please bring back the old view.

**Comment by Moon1moon** [comment]
I'm using Kiwi browser, its a foss browser that supports ublock, desktop
extensions, translate and old tab view

But its not updated freq. as fast as bromite


7 Ara 2022 Çar 19:50 tarihinde bagajohny ***@***.***> şunu
yazdı:
